src/99-appendices/08-coordinate-systems.md

```diff
@@ -195,6 +195,8 @@ Unless specified explicitly in the sidecar file in the
 
 ### Standard template identifiers
 
+All BIDS standard template coordinate system data should be in ``scanner`` coordinates of the T1w template image with units corresponding to `<CoordSysType>CoordinateUnits`.
```

Comment:
```suggestion
When using any of the below templates, coordinates should be in "world" (RAS+) space,
and not voxel indices.
If `<CoordSysType>CoordinateUnits` is not defined or applicable, then the units SHALL
be assumed to be millimeters (mm).
```

Reply:
I was copying off of `ScanRAS` above in same part of the specification:

```
ScanRAS: The origin of the coordinate system is the center of the gradient coil for the corresponding T1w image of the subject, and the x-axis increases left to right, the y-axis increases posterior to anterior and the z-axis increases inferior to superior. For more information see the [Nipy Documentation](https://nipy.org/nibabel/coordinate_systems.html). It is strongly encouraged to align the subject's T1w to ACPC so that the ACPC coordinate system can be used. If the subject's T1w in the BIDS dataset is not aligned to ACPC, ScanRAS should be used.
```

"world" might be more accurate in describing it but from what I've seen in the ieeg part of the spec, scanner has been used more often. Perhaps this should be harmonized.

Reply:
This is from the top of that appendix as well `Please be aware that all data acquisition starts with "device coordinates" (scanner)`